torch/_dynamo/skipfiles.py

```diff
@@ -228,10 +222,12 @@ def _module_dir(m: types.ModuleType):
     "torch.testing",
     "torch.utils._contextlib",
     "torch.utils._pytree",
+    "torch.distributed",
```

Comment:
For @yanboliang

Reply:
Thanks for fixing this! I think you have to remove L225, otherwise, the MacOS tests would be failed.

Reply:
oh, yes, gc gc